calc_si_align.py

```diff
@@ -0,0 +1,219 @@
+import numpy as np
+from Quaternion import Quat
+
+# Baseline ODB_SI_ALIGN used prior to Nov. 2015.  This is expressed to
+# 5 digits of precision and is not an orthonormal rotation matrix.
+
+ODB_SI_ALIGN = np.array([[1.0, 3.3742E-4, 2.7344E-4],
+                         [-3.3742E-4, 1.0, 0.0],
+                         [-2.7344E-4, 0.0, 1.0]])
+
+# Fix the transform matrix by converting to a normalized Quaternion
```

Comment:
This was safeguarding against errors of an uncharacterized size creeping in.  I suspect that given the history of using this for the last 16 years that there is no problem, but just the same I feel better knowing that the matrix is orthonormal to double precision.